JASP-Desktop/components/JASP/Widgets/FileMenu/PrefsResults.qml

```diff
@@ -130,12 +130,41 @@ ScrollView
 					checked:			!preferencesModel.whiteBackground
 					onCheckedChanged:	preferencesModel.whiteBackground = !checked
 					toolTip:			qsTr("This makes the background of all plots transparent, quite useful if you want to use it seamlessly on any background that isn't white.")
-					KeyNavigation.tab:	displayExactPVals
-					KeyNavigation.down:	displayExactPVals
+					KeyNavigation.tab:	resultFont
+					KeyNavigation.down:	resultFont
 				}
 			}
 		}
 
+		PrefsGroupRect
+		{
+			id:		fontGroup
+			title:	qsTr("Font")
+
+			TextArea
+			{
+				id:					resultFont
+				width:				300 * jaspTheme.uiScale
+				height:				150 * jaspTheme.uiScale
+				title:				qsTr("List of Font")
```

Comment:
"List of fonts"

Reply:
Also, can we just do this through a userselectable list of systemfonts instead of them having to type?